api/api.yml

```diff
@@ -165,27 +199,37 @@ paths:
       responses:
         '201':
           description: Application added
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
   /launcher/app-user/cockpit/{id}:
     delete:
-        tags:
-        - launcher
-        summary: Remove application from cockpit
-        operationId: removeApplicationFromCockpit
-        parameters:
-        - in: path
-          name: id
-          schema:
-            type: string
-          required: true
-        responses:
-          '200':
-            description: Application removed
+      tags:
+      - launcher
+      summary: Remove application from cockpit
+      operationId: removeApplicationFromCockpit
+      security:
+      - OAuth2: [appuser]
+      parameters:
+      - in: path
+        name: id
+        schema:
+          type: string
+        required: true
+      responses:
+        '204':
```

Comment:
Mocno bym się zastanowił nad rozdzielaniem tych kodów 200, nie spotkałem się z >201

Reply:
200 musi mieć body. Jeśli nie ma to 204.